docs/nodes/full-storage-node.mdx

```diff
@@ -263,8 +263,8 @@ celestia full start --core.ip http://<ip-address> --keyring.accname <name-of-cus
 
 ### Optional: start the full storage node with SystemD
 
-Follow the tutorial on setting up the full storage node as a background
-process with SystemD [here](./systemd.md).
+Follow the tutorial on
+[setting up Celestia App as a background process with SystemD](./systemd.md).
```

Comment:
[blocking] Since this page is for full storage nodes, I don't think this should state Celestia App. Proposal to do something like:

```suggestion
If you would like to run the full storage node as a background process, follow the [SystemD tutorial](./systemd.md).
```

Reply:
absolutely, thanks Rootul.